src/components/bottomSheet/bottom-sheet.js

```diff
@@ -189,16 +197,17 @@ function MdBottomSheetProvider($$interimElementProvider) {
         $animate.enter(backdrop, options.parent, null);
       }
 
-      var bottomSheet = new BottomSheet(element, options.parent);
-      options.bottomSheet = bottomSheet;
+      if (!options.isLockedOpen) {
+        options.cleanupGestures = registerGestures(element, options.parent);
```

Comment:
Actually after looking again at the PR, can't you move that declaration up to the `isLockedOpen` check and use a `else`? 

```js
if (options.isLockedOpen) {
  options.clickOutsideToClose = false;
  options.escapeToClose = false;
} else {
  options.cleanupGestures = registerGestures(element, options.parent);
}
```